### PR TITLE
Graph 256 - Transaction Handler

### PR DESCRIPTION
**Pull request by kruspy**
[GRAPH-256](https://figmentio.atlassian.net/browse/GRAPH-256)

- Created a new `TendermintTrigger` to process transactions. This allows the `tendermint` subgraphs to include a new type of mapping named `transactionHandler` that will process individual transactions.
- In order to test this a new subgraph can be created using the new mapping. Also, the changes in branch GRAPH-256 for `graph-cli` and `tendermint-protobuf-def` need to be checked out to get the new proto structure `TransactionData`.

### CODE REVIEW

**Review by pienkowb — COMMENTED** [review]
The code looks good 👍  I'm wondering if we could add any validation logic for transaction handlers (just like we did for event handlers) - at this point we could check if there's exactly one transaction handler defined in the manifest.

File: chain/tendermint/src/chain.rs [hunk 1]
```diff
@@ -208,6 +208,14 @@ impl TriggersAdapterTrait<Chain> for TriggersAdapter {
             }))
             .collect();
 
+        triggers.append(
+            &mut shared_block
+                .tx_responses()
+                .cloned()
+                .map(|tx| TendermintTrigger::with_transaction(tx, block.block().clone()))
+                .collect(),
```
Comment (pienkowb) [inline chain/tendermint/src/chain.rs:216]: Was this trailing comma added by a formatter? :)

Comment (kruspy) [reply chain/tendermint/src/chain.rs:216]: Yeah, the formatter made me include it. Should I remove it ?

Comment (pienkowb) [reply chain/tendermint/src/chain.rs:216]: No, it's fine. I believe it's the same formatter that is used across the whole codebase?

Comment (kruspy) [reply chain/tendermint/src/chain.rs:216]: Yeah I guess so. This is the command:

cargo fmt --all -- --check

**Pull request closed without merging**